packages/admin/src/brevoConfiguration/BrevoConfigForm.tsx

```diff
@@ -173,6 +211,18 @@ export function BrevoConfigForm({ scope }: FormProps): React.ReactElement {
                                 name="sender"
                                 label={<FormattedMessage id="cometBrevoModule.brevoConfig.sender" defaultMessage="Sender" />}
                                 fullWidth
+                                required
+                            />
+
+                            <Field
+                                component={FinalFormAutocomplete}
+                                getOptionLabel={(option: Option) => option.label}
+                                isOptionEqualToValue={(option: Option, value: Option) => option.value === value.value}
+                                options={doiTemplateOptions}
+                                name="doiTemplate"
+                                label={<FormattedMessage id="cometBrevoModule.brevoConfig.doiTemplate" defaultMessage="Double opt-in template id" />}
```

Comment:
```suggestion
                                label={<FormattedMessage id="cometBrevoModule.brevoConfig.doiTemplate" defaultMessage="Double Opt-in template id" />}
```

I think that's how it's written, at least that's how I always read it.

Reply:
6795963bd20635bbcb234126a0d77b98dd13a6e5